web/server.js

```diff
@@ -0,0 +1,261 @@
+    destination: function (req, file, cb) {
+      const path='TVB_patients/TVB1/raw/mri'
+      fs.mkdirSync(path, { recursive: true })
+     cb(null,path )
+    },
+    filename: function (req, file, cb) {
+      cb(null, file.originalname )
+    }
+  })
+  
+  var upload1 = multer({ storage: storage }).single('file')
+  
+app.get('/',function(req,res){
+    return res.send('Hello Server')
+})
+app.post('/upload1',function(req, res) {
+    
+    upload1(req, res, function (err) {
+     
+        if (err instanceof multer.MulterError) {
+            return res.status(500).json(err)
+        } else if (err) {
+            return res.status(500).json(err)
+           } 
+          return res.status(200).send(req.file)
+       })
+});
+
+
+
+//For Patient 2
+var storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    const path='TVB_patients/TVB2/raw/mri'
+    fs.mkdirSync(path, { recursive: true })
+    cb(null,path )
+  },
+  filename: function (req, file, cb) {
+    cb(null, file.originalname )
+  }
+})
+
+var upload2 = multer({ storage: storage }).single('file')
+
+app.post('/upload2',function(req, res) {
+  
+  upload2(req, res, function (err) {
+   
+      if (err instanceof multer.MulterError) {
+          return res.status(500).json(err)
+      } else if (err) {
+          return res.status(500).json(err)
+        } 
+      
+      return res.status(200).send(req.file)
+     })
+});
+
+
+
+//For Patient 3
+var storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    const path='TVB_patients/TVB3/raw/mri'
+    fs.mkdirSync(path, { recursive: true })
+    cb(null,path )
+  },
+  filename: function (req, file, cb) {
+    cb(null, file.originalname )
+  }
+})
+
+var upload3 = multer({ storage: storage }).single('file')
+
+app.post('/upload3',function(req, res) {
+  
+  upload3(req, res, function (err) {
+   
+      if (err instanceof multer.MulterError) {
+          return res.status(500).json(err)
+      } else if (err) {
+          return res.status(500).json(err)
+      } 
+      
+      return res.status(200).send(req.file)
+    })
+});
+
+
+
+//For Patient 4
+var storage = multer.diskStorage({
+destination: function (req, file, cb) {
+  const path='TVB_patients/TVB4/raw/mri'
+  fs.mkdirSync(path, { recursive: true })
+  cb(null,path )
+},
+filename: function (req, file, cb) {
+  cb(null, file.originalname )
+}
+})
+
+var upload4 = multer({ storage: storage }).single('file')
+
+app.post('/upload4',function(req, res) {
+
+upload4(req, res, function (err) {
+ 
+    if (err instanceof multer.MulterError) {
+        return res.status(500).json(err)
+    } else if (err) {
+        return res.status(500).json(err)
+    } 
+    
+    return res.status(200).send(req.file)
+  })
+});
+
+
+//Config for Patient 1
+var path1 = __dirname + '/TVB_patients/TVB1/configs';
+fs.mkdirSync(path1, { recursive: true })
+var file1=path1+'/patient_flow.properties'
+app.post("/input1",function(req,res){
+  const data={
```

Comment:
This endpoint could also be generic and receive the current patient as an argument.